tests/queries/0_stateless/02916_to_start_of_interval_origin_overload.sql

```diff
@@ -0,0 +1,12 @@
+set session_timezone = 'UTC';
```

Comment:
Let's shorten the filename to 02916_to_start_of_interval_with_origin.sql/reference.